documentation/docs/source/rst_source/local_server.rst

```diff
@@ -0,0 +1,70 @@
+Running Custom Models
+=================================
+
+|
+
+An example of running a locally served model is provided `here <https://github.com/IBM/ibm-generative-ai/blob/main/examples/user/localserver/localserver.py/>`_. The steps below detail how this is done.
+
+|
+
+1. In a new Python file, import the ibm-generative-ai library and the local server extension.
+
+.. code-block:: python
+
+    from genai.extensions.localserver import CustomModel, LocalLLMServer
+    from genai.model import Model
+    from genai.schemas import GenerateParams, GenerateResult, TokenizeResult, TokenParams
+
+|
+
+2. Create your custom model.
+
+The `CustomModel <https://github.com/IBM/ibm-generative-ai/blob/23f2a42a706b7fb8d99c21da0ddac909528ca1ba/examples/user/localserver/localserver.py#L26-57/>`_ is an Abstract Base Class that the user defines. It requires the following three things:
+
+    * **model_id** - Allows the server to link the model to an incoming request by model ID.
+    * **generate** - The function which generates some output.
+    * **tokenize** - The function which generates tokenize output.
+
+You can view the documentation for CustomModel `here <https://github.com/IBM/ibm-generative-ai/blob/23f2a42a706b7fb8d99c21da0ddac909528ca1ba/src/genai/extensions/localserver/custom_model_interface.py/>`_
```

Comment:
Are we planning to add `CustomModel` to API Docs? If so, should we replace this link to API docs link like the comment I left on `GenerateParams`? @vedem1192

Reply:
@jayolee I asked Veronique about this and it looks like CustomModel isn't going to be added because it's more of an example model for users to adjust.